packages/migrate/src/__tests__/fixtures/seed-from-prisma-config/seed-sqlite-js-extra-args/prisma/seed.js

```diff
@@ -0,0 +1,37 @@
+// Using parseArgs would be better, but it's only available in v18.3.0, v16.17.0 and up
```

Comment:
Why is that a problem? This is in tests and not in user facing code.

Reply:
This is copied verbatim from existing tests. The original comment is from Joel, from this PR from 2 years ago: https://github.com/prisma/prisma/pull/19441